<launch>
  <arg name="launch_roswww" default="true" />
  <include file="$(find roswww)/launch/roswww.launch" if="$(arg launch_roswww)">
    <arg name="port" value="8000" />
  </include>

  <!-- rosbridge_server -->
  <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch"/>

  <param name="language" value="ja-JP"/>
  <param name="candidates_num" value="7"/>
  <node pkg="web_speech_recognition" type="bringup_chrome.py" name="bringup_chrome"/>

  <!-- bringup_server for sobit_mini_head -->
  <node pkg="web_speech_recognition" type="web_speech_recognition_server_for_sobit_mini.py" name="web_speech_recognition_server_for_sobit_mini" output="screen" />

</launch>
